config/locales/de/views/tasks.yml

```diff
@@ -7,6 +7,17 @@ de:
       message_line2: Wenn Sie den/die Benutzer:in zum/zur Co-Autor:in ernennen oder in eine Gruppe einladen möchten, klicken Sie auf den Link unten.
       message_line3: Ansonsten können Sie diese E-Mail ignorieren.
       subject: '%{user} möchte an Ihrer Aufgabe "%{task}" mitarbeiten.'
+    completeness_checklist:
+      checklist: Checkliste
```

Comment:
What about specifying what this checklist is for? Like using `Completeness` (`Vollständigkeit`) or so?